docs/MIR/mir-reporters-template.md

```diff
@@ -7,11 +7,13 @@ This section is a guideline for the **reporter** as they are
 * Make the future owning team think about common issues
 
 * Provide the detail needed by the **reviewer** to decide:
-  *Can this package be well maintained in `main`?* 
+  *Can this package be well maintained in `main`?*
 
 Usage follows {ref}`mir-how-to-use-templates`.
 
-```
+```{code-block} text
+:linenos:
```

Comment:
nitpick (non-blocking): Those line numbers are not static, once the template changes old references to line numbers are broken.

OTOH, the template doesn't change too often and a line number will at least point towards the right context. And it's definitely better than what we have today with no numbers at all.

So +1 from my side.

Reply:
I'm aware and fine with that to be the tradeoff.